code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -1153,6 +1169,18 @@ var/global/list/damage_icon_parts = list() //see UpdateDamageIcon()
 				tail_s.Blend(overlay, ICON_OVERLAY)
 				qdel(overlay)
 
+		if(tail_style.extra_overlay2)
+			var/icon/overlay = new/icon("icon" = tail_style.icon, "icon_state" = tail_style.extra_overlay2)
+			if(wagging && tail_style.ani_state)

```

Comment:
Similar to above, this is doing an icon op that it immediately replaces if a condition is met - could do something similar to above with a ternary condition.